data/migrations/V0091__drop_ofec_sched_b_objects.sql

```diff
@@ -0,0 +1,196 @@
+
+-- ------------------
+-- 	functions and triggers
+-- ------------------
+DROP TRIGGER IF EXISTS nml_sched_b_after_trigger ON disclosure.nml_sched_b;
+DROP TRIGGER IF EXISTS nml_sched_b_before_trigger ON disclosure.nml_sched_b;
+
+DROP TRIGGER IF EXISTS f_item_sched_b_before_trigger ON disclosure.f_item_receipt_or_exp;
+DROP TRIGGER IF EXISTS f_item_sched_b_after_trigger ON disclosure.f_item_receipt_or_exp;
+
+-- trigger functions
+drop function IF EXISTS public.ofec_sched_b_delete_update_queues();
+drop function IF EXISTS public.ofec_sched_b_insert_update_queues();
+drop function IF EXISTS public.ofec_sched_b_update_queues();
+
+-- functions
+drop function IF EXISTS public.ofec_sched_b_update_fulltext();
+drop function IF EXISTS public.ofec_sched_b_update();
+
+drop function IF EXISTS public.drop_old_itemized_schedule_b_indexes(numeric, numeric); 
+
+-- ------------------
+-- Function: public.add_partition_cycles(numeric, numeric)
+-- ------------------
+CREATE OR REPLACE FUNCTION public.add_partition_cycles(
+    start_year numeric,
+    amount numeric)
+  RETURNS void AS
+$BODY$
+
+DECLARE
+
+    first_cycle_end_year NUMERIC = get_cycle(start_year);
+
+    last_cycle_end_year NUMERIC = first_cycle_end_year + (amount - 1) * 2;
+
+    master_table_name TEXT;
+
+    child_table_name TEXT;
+
+    schedule TEXT;
+
+BEGIN
+
+    FOR cycle IN first_cycle_end_year..last_cycle_end_year BY 2 LOOP
+
+        FOREACH schedule IN ARRAY ARRAY['a'] LOOP

```

Comment:
Does it make sense to remove the entire loop and use:
```
 master_table_name = 'ofec_sched_a_master';
```,
etc?

Reply:
The whole function public.add_partition_cycles will be dropped in the next issue that drops ofec_sched_a tables and process, which we plan to implement in next Sprint